macros/utils/data_types/data_type.sql

```diff
@@ -129,3 +138,13 @@
 {% macro bigquery__edr_type_date() %}
     date
 {% endmacro %}
+
+{% macro athena__edr_type_timestamp() %}
+  {%- set config = model.get('config', {}) -%}
+  {%- set table_type = config.get('table_type', 'glue') -%}
```

Comment:
```suggestion
  {%- set table_type = config.get('table_type', 'hive') -%}
```

Reply:
we don't have table_type=glue in Athena, but only hive and iceberg.

Reply:
it's copy/paste from dbt-athena, https://github.com/dbt-athena/dbt-athena/blob/fe44b1cf8796b8681829dab88ad5a35ea079b9b9/dbt/include/athena/macros/utils/timestamps.sql#L12
should it be fixed there as well?

Reply:
ups, you right, I believe that needs to be fix there too.

Reply:
Fixed in dbt-athena too, https://github.com/dbt-athena/dbt-athena/pull/484

Reply:
fixed